.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+name: Bug report
+description: Create a report to help us improve (search for duplicates first)
+title: '[BUG]: '
+labels: bug
+assignees: []
+body:
+
+ - type: checkboxes
+   id: operating-system
+   validations:
+     required: true
+   attributes:
+    label: Operating System
+    description: Which operating system have you used and triggered the issue?
+    options:
+      - label: macOS
+      - label: Microsoft Windows
+      - label: Linux
+      - label: Unix
+      - label: Other (Tell us at the Specifications part!)
+
+ - type: textarea
+   id: specifications
+   attributes:
+    label: Details
+    description: To understand the issue, we need some details of your setup & system!
+    placeholder: |
+      MCreator version: [2021.1 / branch:master / ...];
+      Generator type: [forge-1.15.2 / forge-1.16.5 / ...];
+      OS [Windows / Ubuntu / macOS / ...]; (if not already listed above)
```

Comment:
OS is already above

Reply:
I'd recommend to also move version and generator labels to combo boxes (and add _**any**_ or _**n/a**_ option for e. g. Gradle/UI issues).